### DISCUSSION TITLE
How to set background color or styles of charts?

### DISCUSSION THREAD

**Discussion by nongnapat-s** [PHPOffice/PhpSpreadsheet#2256] [Q&A] [answered]
How to set background color or styles of charts

**Comment by oleibman** [comment]
This was added by PR #3462 on March 19. It is not yet part of an official release, but, if you can test against master, something like the following will work.
```php
$chart->getFillColor()->setColorProperties('FF0000', null, ChartColor::EXCEL_COLOR_TYPE_RGB);
```

**Comment by oleibman** [comment, marked as answer]
PR was included with 1.29.0. Marking question as answered.
